e2e/main_test.go

```diff
@@ -1,11 +1,16 @@
 package e2e
 
 import (
+	"flag"
 	"os"
 	"testing"
 )
 
 func TestMain(m *testing.M) {
+	if !flag.Parsed() {
```

Comment:
This line shouldn't be necessary.

Reply:
I've deleted the line. But I'm not sure do we even need flags at all?
The only flag we defined is [test.log.level](https://github.com/peak/s5cmd/blob/c7dfad601797f1fcd41186cabf1ce1b5ef80fdc4/e2e/util_test.go#L57) and that flag is not used in the Makefile. So it (flagTestLogLevel) takes the default value _err_.  Should I remove them all?

Reply:
Hmm, not sure. In any case, its outside the scope of this PR :)